src/screens/generationList/generationList.scss

```diff
@@ -57,7 +54,11 @@
       @include font-size(m);
     }
   }
-  .transactions {
+  .table-wrapper{
```

Comment:
@artem-rozhkov-dev-optima i've seen this quite often, not that it is important, but the class name and the bracket should be separated by a space (i think its like that almost everywhere), just to keep things more tidy!

Reply:
ok